deployments/helm/nvidia-device-plugin/templates/daemonset-gfd.yml

```diff
@@ -57,7 +57,27 @@ spec:
       {{- end }}
       {{- if $options.hasConfigMap }}
       shareProcessNamespace: true
+      {{- end }}
       initContainers:
+      - image: {{ include "nvidia-device-plugin.fullimage" . }}
+        name: gpu-feature-discovery-init
+        command:  |-
+          #!/bin/bash
+
+          if [[ ! -d /driver-root/etc/nvidia-imex ]]; then
+            echo "No IMEX Nodes config path detected; Skipping"
+            exit 0
+          fi
+
+          mkdir -p /config/etc/nvidia-imex
+          mount /driver-root/etc/nvidia-imex /config/etc/nvidia-imex
```

Comment:
I first thought to rather mount `/driver-root/etc/nvidia-imex` to /config/imex` instead.